.travis.yml

```diff
@@ -6,11 +6,7 @@ php:
 
 # list of paths/bundle to execute
 env:
-    # run setup wizard to install Demo+Content and then test demo with content
   - INSTALL="demoContentNonUniqueDB" PROFILE="demo" TEST="content"
```

Comment:
ah, you refer to all the {% if ezpublish.legacy is defined %} stuff?
If guess you need to add legacy-bridge for now to get the DemoBundle BDD `Content` suite to run.

But you can also add a `Feature/CleanContent` suite with a adjusted version of start_page.feature (from `Content`) , this way we 1. get started on building up that, 2. makes sure the install with no content also works. (and this one we should eventually also run on kernel commits).